README.md

```diff
@@ -115,22 +162,49 @@ Run `isort` and `flake8` checks.
 $ ./run-code-checks.sh
 ```
 
-#### Use git pre-commit hooks
+### UI tests (unit and ui-tests)
+
+```bash
+$ cd ui
+$ yarn test # in ui folder
+```
 
-Use `pre-commit`
+### e2e
 
 ```bash
-$ poetry run pre-commit install
+$ ./run-e2e.sh
 ```
 
-### How to login
+---
 
-For developement, we are using local login for our service.
+## How to import records
 
-#### UI
+There is a command `inspirehep importer records` which accepts url `-u`, a directory of `JSON` files `-d` and `JSON` files `-f`.
+A selection of demo records can be found in `data` directory and they are structure based on the record type (i.e. `literature`). Examples:
 
-For the UI you can login http://localhost:8081/user/login/local
+#### With url
 
-#### Backend
+```bash
+# Local
+$ poetry inspirehep importer records -u https://labs.inspirehep.net/api/literature/20 -u https://labs.inspirehep.net/api/literature/1726642
```

Comment:
You should mention that with this setup it's also required to run worker to properly import and index a record.